docs/testing/automated_benchmark_test.md

```diff
@@ -1,8 +1,8 @@
 # When to perform automated benchmarking tests
 Automated benchmarking tests are additional functional and performance tests to the existing TAP device-based ones. This test suit relies on a configured environment including hypervisors and started VMs, as well as configured SSH authorized key of the execution machine starting the benchmarking tests. In the end, running these benchmarking tests is useful for verifying if dpservice works correctly together with actual running VMs for both offloading and non-offloading modes. It also verifies if networking performance meets specified values during dpservice development.
 
-# Required hypervisor and VM setup
-To successfully run these automated benchmarking tests, currently, 2 hypervisors and 3 VMs need to be prepared beforehand, especially putting the ssh key of the machine executing the benchmarking tests into the above mentioned hypervisors and VMs.
+# Required hypervisor setup
+To successfully run these automated benchmarking tests, currently, 2 hypervisors and 3 VMs need to be prepared beforehand, especially putting the ssh key of the machine executing the benchmarking tests into the above mentioned hypervisors.
 
 ## Prerequisite
 
```

Comment:
Maybe also mentioning the need to have the ssh key pair ready under .ssh and the need for a valid dp_service.conf file on hypervisors to be used.